README.md

```diff
@@ -53,45 +50,48 @@ If you are having any issues running the SDK in the demo app, please create a qu
 *  [iOS](https://htmlpreview.github.io/?https://github.com/klarna/klarna-mobile-sdk/blob/master/docs/ios/index.html)
 *  [Android](https://htmlpreview.github.io/?https://github.com/klarna/klarna-mobile-sdk/blob/master/docs/android/klarna-mobile-sdk/index.html)
 
+## Support
+If you are having any issues using the SDK in your project or if you think that something is wrong with the SDK itself, please create an issue on [Github](https://github.com/klarna/kp-android-example-app/issues) or report a bug by following the guidelines in the next section. 
+
 ## Contributing
-Thank you for reading this and taking time to contribute to Klarna In-App SDK - Android Demo App! Below is a set of guidelines to help you contribute whether you want to report a bug, come with suggestions or modify code.
+Thank you for reading this and taking the time to contribute to Klarna In-App SDK - Android Demo App! Below is a set of guidelines to help you contribute whether you want to report a bug, come with suggestions or modify code.
 
 ### How can I contribute?
 #### Reporting Bugs
 This section will guide you through submitting a bug report for Klarna In-App SDK - Android Demo App.
 
-Before submitting a bug report, please check that the issue hasn't been reported before. If you find a *Closed* issue that seem to describe an issue that is similar to what you want to report, open a new issue and link to the original issue in the new one. When you have checked that the issue hasn't been reported before **fill out [the required template](https://github.com/klarna/kp-android-example-app/blob/ca76e619a15c7646bfb39f4cf587594146eeebfb/.github/ISSUE_TEMPLATE/bug_report.md)** which will help us resolve the issue faster. 
+Before submitting a bug report, please check that the issue hasn't been reported before. If you find a *Closed* issue that seems to describe an issue that is similar to what you want to report, open a new issue and link to the original issue in the new one. When you have checked that the issue hasn't been reported before, please **fill out [the required template](https://github.com/klarna/kp-android-example-app/blob/ca76e619a15c7646bfb39f4cf587594146eeebfb/.github/ISSUE_TEMPLATE/bug_report.md)** which will help us resolve the issue faster. 
```

Comment:
https://github.com/klarna/kp-android-example-app/blob/ca76e619a15c7646bfb39f4cf587594146eeebfb/.github/ISSUE_TEMPLATE/bug_report.md

Maybe change to master?

Reply:
@polar133 Good point!